.github/workflows/build.yml

```diff
@@ -102,15 +112,13 @@ jobs:
             echo "json<<$EOJSON"
             cat <<<"$json"
             echo "$EOJSON"
-          } >> "$GITHUB_ENV"
+          } >> "$GITHUB_ENV" >> "$GITHUB_OUTPUT"
```

Comment:
@tianon here is the bit that forwards the json to other jobs (with line 48 above)

Reply:
😕 but the `Sign` step is unchanged; should it be adjusted to use the `json` from the output instead of the `env`?

Can the env creation then be removed? Ah, no, the `Check` step still uses it.

Reply:
Oh, this is just shell, so ` >> "$GITHUB_ENV" >> "$GITHUB_OUTPUT"` won't work as desired. This'll redirect `stdout` to the GitHub env file and then change to the GitHub output file before any output is sent, so the env won't get anything. It needs to swap to `tee` to output to more than one thing:

```bash
} | tee -a "$GITHUB_ENV" "$GITHUB_OUTPUT" >/dev/null
```

Reply:
Really? Because it worked locally and on the test repo

Reply:
😕 Are you using a different shell locally? That's not what I get with a bash container.

```console
$ docker run -it --rm bash
bash-5.2# echo "foo" > file1
bash-5.2# echo "bar" >> file1 >> file2
bash-5.2# cat file1
foo
bash-5.2# cat file2
bar
bash-5.2#
```

Reply:
I think your test workflow worked because none of the later steps use the json because they are the already expanded commands.

Reply:
Oh, I guess I am using zsh not bash

Reply:
It weird that ir works in the test repo because we do use the json later in the process. I will update to this since it is safer

Reply:
Removed this part